checker/tests/resourceleak/ACSocketTest.java

```diff
@@ -360,10 +360,13 @@ void createNewServerSocket(InetSocketAddress address, boolean b, boolean c) thro
     ServerSocket socket;
 
     if (b) {
+      // :: error: required.method.not.called
```

Comment:
Not sure why these new errors come up from this PR?

Reply:
At line 374, there's a `bind()` method call that could potentially throw an exception, leaving these resources open in that scenario.